PowerShellTools.HostService/ServiceManagement/Debugging/PowerShellDebuggingService.cs

```diff
@@ -931,9 +1057,9 @@ public IEnumerable<CallStack> GetCallStack()
                             (string)psFrame.FunctionName.ToString(),
                             (int)psFrame.ScriptLineNumber));
                 }
-                else if(GetDebugScenario() == DebugScenario.LocalAttach)
+                else if(GetDebugScenario() == DebugScenario.RemoteAttach || GetDebugScenario() == DebugScenario.LocalAttach)

```

Comment:
Is it possible to just store the results of GetDebugScenario in a local var? [or do we expect it to change during these if checks?]